qiskit/pulse/pulse_lib/sample_pulse.py

```diff
@@ -21,13 +21,15 @@
 from ..channels import PulseChannel
 from ..exceptions import PulseError
 from .pulse import Pulse
+warnings.warn("the SamplePulse class is deprecated", DeprecationWarning,
```

Comment:
Unfortunately, I don't think we can leave this here, because this will then print for any user who imports anything from qiskit. This is because `qiskit.__init__` traverses other `__init__` files until it eventually imports `qiskit.pulse.__init__`, and that leads to this line

Reply:
ok will remove